microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+
+The Settings menu now provides options for changing DevTools between docked locations and a separate window. This menu also contains settings for how the Activity Bar is displayed. Starting in Edge 105, these options can be navigated and changed more quickly. The previous submenus have been replaced with toggle buttons. Additionally, improved contrast means the option you have selected is more easily visible at a glance.
+
+Before:
+
+![1 before](./devtools-105-images/before-docking-menu.msft.png)
+
+After:
+
+![1 after](./devtools-105-images/after-docking-menu.msft.png)
+
+<!-- 
+Screenshot 1, "before" in Edge 104: Ensure Focus Mode is enabled. Open Settings menu, then navigate to the docking submenu to show options.
+Screenshot 1, "after" in Edge 105: Ensure Focus Mode is enabled. Open Settings menu to show DevTools docking location and Activity Bar location settings. -->
+
+You can now change the orientation of the Quick View panel as well. To change Quick View to a vertical split, select the location toggle (**Dock Quick View to the right**). Selecting this toggle a second time will return Quick View to a horizontal split.
+
+![2](./devtools-105-images/quickview-console.png) 
+
+<!-- 
+Screenshot 2: Ensure Focus Mode is enabled. With Elements open in main panel and Console in Quick View, press the location toggle at the top right of Quick View to change its orientation to vertical. -->
+
+To minimize Quick View, use the **Collapse** toggle or press `Escape`:
+
+![3](./devtools-105-images/focus-mode-improved-location-controls3.png)
+<!-- Screenshot 3: Continuation of previous screenshot. Minimize Quick View in vertical orientation with with Esc keyboard shortcut. See below. -->
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+<!-- ====================================================================== -->
+
+## Fix: Search in the Sources and Network tools now works as expected in Focus Mode
+
+<!-- Rank 4 -->
+<!-- Title: Fix: Search in the Sources and Network tools now works as expected in Focus Mode  -->
+<!-- Subtitle: Try Focus Mode, a new, more simplified, and streamlined UI for DevTools!-->
+
+[Focus Mode experiment](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode#enable-focus-mode) in previous versions of Microsoft Edge didn't properly showcase the Search feature in the Network tool nor the *Search in all files* feature in the Sources tool. In Microsoft Edge version 105, these issues have been fixed.
+
+You can now search in the Network tool via the Search icon.  
+
+![1](./devtools-105-images/networking-search.png)
+
+You can leverage the *Search in all files* feature in the Sources tool by opening the context menu on files in the Page section.  This will display the Search tool in the Quick View.
```

Comment:
```suggestion
You can leverage the *Search in all files* feature in the **Sources** tool by opening the context menu on files in the **Page** section.  This will display the **Search** tool in the **Quick View**.
```

Reply:
i pulled then incorp'd